### ISSUE TITLE
Status of this SDK

### ISSUE THREAD

**Issue by wrs** [salesforce-marketingcloud/FuelSDK-Ruby#87]
We are trying to bring up our MarketingCloud integration, and trying to decide whether to use this SDK, which seems to be abandoned. @amagar088, you did the last string of commits (ending in 2016) — are you still around? What is the roadmap? Thanks.

**Comment by hatch-carl** [comment]
Don't use it. There are still extant bugs and the code is old, weird, and convoluted.

It requires some more bootstrapping, but I have been able to use Savon with the WSDL to make up for the deficiencies in this SDK and will probably continue to reduce dependence on this SDK moving forward.

**Comment by wrs** [comment]
Yes, we made our own library based on Savon rather than using the SDK at all.

**Comment by wrs** [comment]
@hendricius It's not, but it could be…it's only implements the few calls we use, though, it's not a general SDK.

**Comment by hendricius** [comment]
@wrs thanks :)

**Comment by popcorn** [comment]
@wrs - hey Walter, I see you mentioned your client could be open source.

Is it open somewhere? I'm having huge trouble implementing the SFMC SOAP client and it would be of great help to me to see how other people have done it.
